wyvern/components/features/feature_store.py

```diff
@@ -69,7 +73,9 @@ async def fetch_features_from_feature_store(
         )
 
         if response.status_code != 200:
-            logger.error(f"Error fetching features from feature store: {response}")
+            logger.error(
+                f"Error fetching features from feature store: [{response.status_code}] {response.json()}",
```

Comment:
i usually do `response.text/content` instead of json() cuz json() gets busted if the response content is not in the json format.

Reply:
I'll update, thanks for the heads up. I didn't know 😅